### PR TITLE
Adding reset password feature

### PR DESCRIPTION
**Pull request by gupta-arpit**

### CODE REVIEW

**Comment by akshat2602** [comment]
In line 262 of authActions.js, the if condition should check for status code 204 as that is what djoser is returning by default. You can modify it as follows:
`if (res.status >= 200 && res.status < 304)`

File: eda-frontend/src/redux/actions/authActions.js [hunk 1]
```diff
@@ -232,3 +242,37 @@ export const googleLogin = (host, toUrl) => {
...
+          payload: {
+            data: 'Successfully Signed Up! A verification link has been sent to your email account.'
+          }
+        })
+        // history.push('/login')
+      }
+    })
+    .catch((err) => {
+      var res = err.response
+      if ([400, 401, 403, 204, 304].includes(res.status)) {
```
Comment (gupta-arpit) [inline eda-frontend/src/redux/actions/authActions.js:274]:
```suggestion
      if (res.status >= 200 && res.status < 304) {
```

**Comment by gupta-arpit** [comment]
> In line 262 of authActions.js, the if condition should check for status code 204 as that is what djoser is returning by default. You can modify it as follows:
> `if (res.status >= 200 && res.status < 304)`

done.

File: eda-frontend/src/pages/ResetPassword/Confirmation.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+}))
+
+export default function ResetPasswordConfirm ({ match }) {
+  const classes = useStyles()
+
+  const auth = useSelector(state => state.authReducer)
+
+  const dispatch = useDispatch()
+  var homeURL = `${window.location.protocol}\\\\${window.location.host}/`
+  console.log("hey")
```
Comment (gupta-arpit) [inline eda-frontend/src/pages/ResetPassword/Confirmation.js:48]:
```suggestion
```

**Pull request merged**